src/asm/parser.y

```diff
@@ -277,7 +277,7 @@ static void strfmt(char *dest, size_t destLen, char const *fmt, size_t nbArgs, s
 			a++;
 			continue;
 		} else if (a == nbArgs) {
-			error("STRFMT: Not enough arguments for format spec\n", a + 1);
+			error("STRFMT: Not enough arguments for format spec\n");
```

Comment:
This should only be reported once, and report the expected / actual amounts. (+ test case)

Reply:
It was reported only once, there is a test for this. But I changed how the only once is implemented now, and reported the amount of got/needed arguments.

Reply:
You're right, I actually missed the lines below, and assumed it worked the same as in the lexer. The new change is much better, though.